OrbitGl/App.cpp

```diff
@@ -520,7 +524,7 @@ ErrorMessageOr<void> OrbitApp::OnSavePreset(const std::string& filename) {
 
 //-----------------------------------------------------------------------------
 ErrorMessageOr<void> OrbitApp::ReadPresetFromFile(const std::string& filename,
-                                                  Preset* preset) {
+                                                  PresetInfo* preset_info) {
```

Comment:
I think that function should return a `ErrorMessageOr<PresetInfo>` instead of having an out-parameter.

Reply:
Fixed it, thanks!